src/edfi-performance-test/edfi_performance_test/performance_tester.py

```diff
@@ -26,6 +27,14 @@ def prepare_for_output(args: MainArguments) -> str:
     return output_base_path
 
 
+def monitor_fail_ratio(runner: Runner):
+    while runner.state not in [STATE_STOPPING, STATE_STOPPED, STATE_CLEANUP]:
+        time.sleep(1)
+        if runner.stats.total.fail_ratio > 0.5:
```

Comment:
I like this, and it works great in practice. One small request before merging the code: can you add a note in the readme stating that the tester will quit if it experiences errors on more than 50% of requests?

Reply:
done